packages/gucharmap.rb

```diff
@@ -20,6 +20,8 @@ class Gucharmap < Package
   depends_on 'itstool'
```

Comment:
Update to [12.0.1](https://download.gnome.org/sources/gucharmap/12.0/gucharmap-12.0.1.tar.xz)?